### ISSUE TITLE
`runtime.setSkillManifest is not a function` error when starting a new Basic Assistant project

### ISSUE THREAD

**Issue by darrenj** [microsoft/BotFramework-Composer#6823]
## Describe the bug

When starting a newly created Basic Assistant project Composer reports the following error: `runtime.setSkillManifest is not a function`

```
Start bot failed

Message:
runtime.setSkillManifest is not a function
Stack Trace:
TypeError: runtime.setSkillManifest is not a function
    at LocalPublisher2.publishAsync (/Applications/Bot Framework Composer.app/Contents/Resources/app.asar.unpacked/extensions/localPublis
```

## Version

1.4.0.nightly-233315

## Browser

- [x] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [x] macOS
- [ ] Windows
- [ ] Ubuntu

## To Reproduce

1. Create New Basic Assistant
2. Add LUIS/QnA Key
3. Start Bot

## Expected behavior

Project starts succesfully

## Screenshots

![image](https://user-images.githubusercontent.com/33519520/114164220-f3041180-9922-11eb-8c5e-d6a33bf6cfd5.png)

**Comment by rvinothrajendran** [comment]
When starting a new Blank bot also look like same issue

Start bot failed

Message:
runtime.setSkillManifest is not a function
Stack Trace:
TypeError: runtime.setSkillManifest is not a function
    at LocalPublisher2.publishAsync (....\Bot Framework Composer\resources\app.asar.unpacked\extensions\localPublish\dist\extension.js:30241:25

**Comment by tracyboehrer** [comment]
@darrenj I don't think this is a Runtime issue, but I'll follow up with it.

**Comment by benbrown** [comment]
This was resolved by #6816
